crates/matrix-sdk-base/src/store/traits.rs

```diff
@@ -392,12 +392,13 @@ pub trait StateStore: AsyncTraitDeps {
         room_id: &RoomId,
     ) -> Result<Vec<QueuedEvent>, Self::Error>;
 
-    /// Updates the send queue wedged status for a given send queue event.
+    /// Updates the send queue error status (wedge) for a given send queue
+    /// event.
```

Comment:
Can you clarify the contract around what it means when `error` is not set? (I suppose that means "unwedge", but it's not specified here, and not obvious at first sight)